DEMS-Common/src/UDP/OperationCode.java

```diff
@@ -52,7 +52,13 @@ of this software and associated documentation files (the "Software"), to deal
     NO_RESP_NOTIFICATION(1201),
     ACK_NO_RESP_NOTIFICATION(3201),
     FAULY_RESP_NOTIFICATION(1202),
-    ACK_FAULY_RESP_NOTIFICATION(3202);
+    ACK_FAULY_RESP_NOTIFICATION(3202),
+	
+	//For RM-RE
+	RESTORE_ORDER_NOTIFICATION(1666),
```

Comment:
Restore -> removes data and add back data, Restart -> simply restart the replica

'to five all' ???

Reply:
when u restart you will need to add the data back too. It will be the RM who asks the sequencer for all the requests and execute them one by one on his replica

Reply:
although the more i'm thinking it might be easier to get the map from another instance and just copy it =?